src/controllers/graphql/user-controller.js

```diff
@@ -11,3 +13,17 @@ export const deleteUser = (_, id, { models }) => {
     where: id
   })
 }
+
+export const signup = async (_, { data: user }, { models, secret }) => {
+  try {
+    const newUser = await models.user.create(user)
+    const token = jwt.sign({ sub: newUser.id }, secret, { expiresIn: '10d' })
+
+    return {
+      user: newUser,
+      jwt: token
+    }
+  } catch (error) {
+    return error
```

Comment:
Errors should not be _returned_ but _thrown_.

Reply:
When I put *throw* instead of *return* StandardJS says that the try/catch wrapper is unnecessary, so I will try deleting it and see if everything works fine. Thanks!